src/domain/communication/message/message.service.ts

```diff
@@ -1,31 +0,0 @@
-import { LogContext } from '@common/enums/logging.context';
```

Comment:
@valeksiev  you may want to add this somewhere else, when sending the VC message to the matrix room